bitcoin/src/blockdata/opcodes.rs

```diff
@@ -34,636 +34,299 @@ pub struct All {
     code: u8,
 }
 
-// private import so we don't have to use `all::OP_FOO` in this file.
-use self::all::*;
+macro_rules! all_opcodes {
+    ($($op:ident => $val:expr, $doc:expr);*) => {
+        // private import so we don't have to use `all::OP_FOO` in this file.
+        use self::all::*;
+
+        /// Enables wildcard imports to bring into scope all opcodes and nothing else.
+        ///
+        /// The `all` module is provided so one can use a wildcard import `use bitcoin::opcodes::all::*` to
+        /// get all the `OP_FOO` opcodes without getting other types defined in `opcodes` (e.g. `All`, `Class`).
+        ///
+        /// This module is guaranteed to never contain anything except opcode constants and all opcode
+        /// constants are guaranteed to begin with OP_.
+        pub mod all {
+            use super::All;
+                $( 
+                    /// $doc
```

Comment:
Unfortunately, this doesn't work. If I'm not mistaken `#[doc = $doc]` should work instead.